eiffel-vocabulary/EiffelArtifactDeployedEvent.md

```diff
@@ -0,0 +1,19 @@
+
+# EiffelArtifactDeployedEvent (ArtD)
+
+The EiffelArtifactDeployedEvent declares that a software artifact (declared by EiffelArtifactCreatedEvent) has been deployed and is consequently available for use.
+
+## Data Members
+### data.reason
+__Type:__ String  
+__Required:__ No  
+__Description:__ The location at which the artifact is deployed.
```

Comment:
Hi Daniel, you are correct this does not make sense. This work I have done is essentially a dump of any data to get me started ( a skeleton of an event) . Any data in these events is probably wrong and lots of data is obviously missing.  my intent was to get something "in" the events to get me started that's all.